app/Main.hs

```diff
@@ -61,6 +68,13 @@ baseDaoStorageParser = do
     (#help .! "Administrator of the BaseDAO contract")
```

Comment:
Hm, if we change `the BaseDAO contract` to just `a DAO contract`, probably we'll be able to use this parser for both DAOs, right? I'd like to do it because we may (and will) have more shared fields configurable and thus more code will be copy-pasted.